problem-spring-web/src/main/java/org/zalando/problem/spring/web/advice/general/ResponseStatusAdviceTrait.java

```diff
@@ -0,0 +1,30 @@
+package org.zalando.problem.spring.web.advice.general;
+
+import org.apiguardian.api.API;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.server.ResponseStatusException;
```

Comment:
I think the class doesn't exist in Spring 4

Reply:
Looks like a good opportunity to drop spring 4 support.